coveralls/api.py

```diff
@@ -108,15 +108,23 @@ def load_config_from_github(self):
         # the same -- forceibly using Github's flow seems to be more stable
         self.config['repo_token'] = os.environ.get('GITHUB_TOKEN')
 
+        # TODO: Clarify default service.
+        # From experiments, it seems that service should always be set to
+        # 'github'. However, that is not yet confirmed - for discussion, see
+        # https://github.com/lemurheavy/coveralls-public/issues/1710.
+        # For compatibility, the service continues to be set to 'github-actions'
+        # for now. Users can specify the service name manually to override
+        # this.
+        service = 'github-actions'
+
+        job = os.environ.get('GITHUB_RUN_ID')
```

Comment:
Can we change job ID to be `GITHUB_JOB` it should be unique within a workflow, so it's better use it for different jobs.

```suggestion
        job = os.environ.get('GITHUB_JOB')
```

Reply:
`GITHUB_JOB` is the **name** of the workflow, not its ID.
 
 `GITHUB_RUN_ID` is an ID that is unique within Actions, but the same for all Actions jobs in the same workflow run. That allows coveralls.io to combine the jobs of a build.